mlflow/tracing/provider.py

```diff
@@ -1,18 +1,30 @@
+import logging
 from typing import Optional
 
 from opentelemetry import trace
 from opentelemetry.sdk.trace import TracerProvider
 from opentelemetry.sdk.trace.export import SimpleSpanProcessor
 from opentelemetry.util._once import Once
 
+from mlflow.entities.trace_info import TraceInfo
+from mlflow.entities.trace_status import TraceStatus
+from mlflow.exceptions import RestException
 from mlflow.tracing.clients import TraceClient, get_trace_client
 from mlflow.tracing.export.mlflow import MlflowSpanExporter
+from mlflow.tracing.utils import format_trace_id
+from mlflow.utils.databricks_utils import (
+    is_in_databricks_model_serving_environment,
+    is_in_databricks_runtime,
+)
 
 # Once() object ensures a function is executed only once in a process.
 # Note that it doesn't work as expected in a distributed environment.
 _TRACER_PROVIDER_INITIALIZED = Once()
 
 
+_logger = logging.getLogger(__name__)
+
+
 def get_tracer(module_name: str):
```

Comment:
Shall we make this private, since it's no longer called elsewhere?

Reply:
Absolutely!